### PR TITLE
Create fake nurses and devices via Drush

### PR DESCRIPTION
**Pull request by AronNovak**
#2969

### CODE REVIEW

**Comment by AronNovak** [comment]
image
image

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc [hunk 1]
```diff
@@ -15,10 +17,56 @@ function hedley_migrate_drush_command() {
...
+            'value' => '000000' . sprintf('%04d', $i),
+          ],
+        ],
+      ],
+    ]);
+    node_save($device);
+    $nurse = entity_create('node', [
+      'type' => 'nurse',
+      'title' => 'Nurse ' . $i,
+      'field_pin_code' => [
```
Comment (anvmn) [inline server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc:57]: Maybe use `hedley_user_get_unique_pincode()`?

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc [hunk 2]
```diff
@@ -15,10 +17,56 @@ function hedley_migrate_drush_command() {
...
 
+/**
+ * Creates fake nurses and devices.
+ */
+function hedley_migrate_create_dummy_nurses_devices($amount = 100) {
+  for ($i = 0; $i < $amount; $i++) {
+    $device = entity_create('node', [
+      'type' => 'device',
+      'title' => 'Device ' . $i,
+      'field_pairing_code' => [
```
Comment (anvmn) [inline server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc:45]: Maybe use `hedley_device_get_unique_pairing_code()`?

**Review by anvmn — COMMENTED** [review]
@AronNovak

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc (hunk 1 above)
Comment (AronNovak) [reply server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc:57]: :+1:

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc (hunk 2 above)
Comment (AronNovak) [reply server/hedley/modules/custom/hedley_migrate/hedley_migrate.drush.inc:45]: Done

**Comment by AronNovak** [comment]
^^ @anvmn

**Comment by AronNovak** [comment]
^ @anvmn

**Review by anvmn — APPROVED** [review]

**Pull request merged**